emloop/tests/cli/common_test.py

```
@@ -283,12 +285,12 @@ def test_create_model(tmpdir):
     assert isinstance(restored_model, DummyModelWithKwargs)
 
     # test restoring when the model class is not found
-    new_config = deepcopy(config)
-    new_config['model']['class'] = 'nonexistingmodule.IDontExist'
-    new_config['model']['restore_fallback'] = 'emloop.tests.cli.common_test.DummyModelWithKwargs2'
-    with pytest.raises(ImportError):
-        create_model(config=new_config, output_dir=tmpdir + '_restored', dataset=dataset,
-                                      restore_from=tmpdir)
+#    new_config = deepcopy(config)
```

Either fix it, let it fail or delete the test.